src/pages/components/split-button.mdx

```diff
@@ -39,6 +39,40 @@ components:
 
 ## How to use it
 
+### Default
+
+The typical usage of a Split Button component.
+
+import SplitButtonDefault from '../../examples/button/SplitButtonDefault.tsx';
+import SplitButtonDefaultCode from '!!raw-loader!../../examples/button/SplitButtonDefault.tsx';
+
+<CodeExample code={SplitButtonDefaultCode}>
+  <SplitButtonDefault />
+</CodeExample>
+
+### Danger
+
+Use danger styling for Split Buttons that enable destructive action.
+
+import SplitButtonDanger from '../../examples/button/SplitButtonDanger.tsx';
+import SplitButtonDangerCode from '!!raw-loader!../../examples/button/SplitButtonDanger.tsx';
+
+<CodeExample code={SplitButtonDangerCode}>
+  <SplitButtonDanger />
+</CodeExample>
+
+### Disabled
+
+A disabled Split Button prevents user interaction. It doesn’t appear in the tab order,
+can’t receive focus, and may not read aloud by a screenreader.
```

Comment:
Proving that the human brain can insert missing words in literally any scenario, it appears that when I rewrote the disabled copy, I omitted the word "be" in the last line. I'll open a PR fixing this.

Reply:
Actually, I'm hitting all kinds of roadblocks to get my branch created, yarn started etc. Could you please do this for me? the sentence should read:

> A disabled Split Button prevents user interaction. It doesn’t appear in the tab order,
can’t receive focus, and may not **_be_** read aloud by a screenreader.

Reply:
Wow, that's nuts. I missed it too. Will fix